cmake/FindGzOGRE2.cmake

```diff
@@ -75,33 +76,18 @@ if (${GzOGRE2_FIND_VERSION_MAJOR})
   endif()
 endif()
 
+function(colcon_do_nothing)
```

Comment:
Out of curiosity, how this works when using colcon?

Reply:
Colcon is pretty naive when it comes to finding packages.  It does a regex across cmake files looking for `find_package(foo)` to see if there is a dependency on foo.

This forces colcon to understand that ogre-next needs to come before gz-cmake if you are building both in the same workspace.

The alternative is to use `colcon.pkg` or `package.xml`, but I don't think that we have made any progress on that front yet. I can add a todo to remove this when we do something with metadata.

Reply:
Thanks for the pointer, I think it is fine to add an small comment like this one in the source code by now until we resolve the approaches of package.xml or colcon.pkg.